scripts/config-list.js

```diff
@@ -58,6 +60,24 @@ mc.getAsync('limits')
       else {
         output.allowedIPs = data
       }
+      return mc.getAsync('allowedEmailDomains')
+    }
+  )
+  .then(
+    function (data) {
+      if (!data) { console.error('no allowedEmailDomains set') }
```

Comment:
@vbudhram I pushed a second commit that fixes this to be consistent with our usual style.